automation-tests/test/specs/debugApexTests.e2e.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
```

Comment:
Since @CristiCanizales is actively working on this file, I'll leave the copyright alone.  @CristiCanizales , when you check this in, please update the copyright to 2023.